index.js

```diff
@@ -68,26 +69,24 @@ const Minesweeper = {
 				}
 			}
 		}
+
+		if (this.unopened <= 0) {
+			this.submit()
+		}
 	}
 
 	, flag: function () {
 		const x = this.cursorX
 		const y = this.cursorY
-		const isFlagged = !this.flagged[x][y]
-		this.flagged[x][y] = isFlagged
-		if (this.mines[x][y]) this.flaggedMines += isFlagged ? 1 : -1
-		if (this.flaggedMines >= this.nrOfMines) return this.submit()
+		if (!this.opened[x][y]) {
+			this.flagged[x][y] = !this.flagged[x][y]
+		}
 	}
 
 
 
 	, abort: function () {
-		this.done = this.aborted = true
-		for (let y = 0; y < this.mines.length; y++) {
-			for (let x = 0; x < this.mines[0].length; x++) {
-				this.opened[x][y] = true
-			}
-		}
+		this.aborted = true
```

Comment:
Put in `this.done` again.

Reply:
Do you care if I keep `this.done` false here, or do you want it to be true like before? Setting it to true will require changing the render function again.

Reply:
Keep it to `true` to keep it compatible with my other prompts. This is not documented anywhere, but I'd like `prompt-skeleton` to rely on it in the future.